docker-compose.yaml

```diff
@@ -22,5 +22,5 @@ services:
       - 1234:1234
     environment:
       API_URL: http://api:3000
-      NEXT_PUBLIC_API_URL: http://localhost:3000
+      NEXT_PUBLIC_API_URL: http://localhost:1234/crabfit
```

Comment:
What does this mean ? The API is located a `http://localhost:3000` in the dev environment, `http://localhost:1234/crabfit` is the url of the frontend.

Reply:
yep my bad I revert this.